xbmc/pvr/recordings/PVRRecordings.cpp

```diff
@@ -194,17 +196,28 @@ int CPVRRecordings::GetNumRecordings()
   return m_recordings.size();
 }
 
-int CPVRRecordings::GetRecordings(CFileItemList* results)
+bool CPVRRecordings::HasDeletedRecordings()
+{
+  CSingleLock lock(m_critSection);
+  return m_bHasDeleted;
+}
+
+int CPVRRecordings::GetRecordings(CFileItemList* results, bool deleted)
 {
   CSingleLock lock(m_critSection);
 
+  int recCount = 0;
   for (PVR_RECORDINGMAP_CITR it = m_recordings.begin(); it != m_recordings.end(); it++)
   {
+    if (it->second->m_isDeleted != deleted)
```

Comment:
we should use the getter `IsDeleted()` instead of the member here and everywhere else.